collaboration-service/common/constants.js

```diff
@@ -0,0 +1,11 @@
+let uri;
+
+if (process.env.NODE_ENV == "production") {
```

Comment:
Hmm if someone env file has NODE_ENV on production, this could break?

Reply:
For local/production testing, they will need to start redis-server before running the code. (E.g., redis-server command in ubuntu)